app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala

```diff
@@ -70,4 +71,9 @@ class APIDocumentationController @Inject()(cc: ControllerComponents,
       assets.at(s"/public/api/conf/$version", file)(request)
     }
 
+  def yaml(version: String, file: String): Action[AnyContent] =
```

Comment:
🚫 The same function already exists in this class, also called yaml

Reply:
have resolved the conflict.please check

Reply:
this is for first publish with both yaml and raml. we need this function.

Reply:
![image](https://user-images.githubusercontent.com/11063742/221882982-3e6f4e1e-8d97-48e8-800a-cd3f637095ac.png)

We need it but we have two of them here